image/templates/helm/stackrox-central/internal/config-shape.yaml

```diff
@@ -94,6 +115,12 @@ customize:
     podLabels: {} # dict
     podAnnotations: {} # dict
     envVars: {} # dict
+  centralDB:
```

Comment:
scanner's db is `scanner-db`. The helm chart is a bit inconsistent already, but let's try to keep it as consistent as possible here.

```suggestion
  central-db:
```

Reply:
renamed to db: